<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsEditorCommon</class>
 <widget class="QWidget" name="SettingsEditorCommon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/icons.qrc">
    <normaloff>:/icons/48x48/settings.png</normaloff>:/icons/48x48/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupRadio">
     <property name="title">
      <string>Radio</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelRadio">
        <property name="text">
         <string>Default radio category:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editRadio"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupUdpxy">
     <property name="title">
      <string>UDPXY support</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkUdpxy">
        <property name="text">
         <string>Enable UDPXY support</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editUdpxyUrl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="editUdpxyPort">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>1234</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelUdpxyUrl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Router IP or name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelUdpxyPort">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Router port:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupGui">
     <property name="title">
      <string>Interface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelToolbarStyle">
        <property name="text">
         <string>Toolbar style:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboToolbarLook">
        <property name="currentIndex">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>Icon only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text beside icon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text under icon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>System default</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkSplash">
        <property name="text">
         <string>Show splash screen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../data/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelUdpxyUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelUdpxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUdpxyUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUdpxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
